diff --git a/src/Schema/Invoice/InvoiceLine.php b/src/Schema/Invoice/InvoiceLine.php
index 26761c8..609232a 100644
--- a/src/Schema/Invoice/InvoiceLine.php
+++ b/src/Schema/Invoice/InvoiceLine.php
@@ -72,7 +72,10 @@ class InvoiceLine implements Arrayable
 	#[Map('LineExtensionAmountCurr')]
 	private ?string $lineExtensionAmountCurr = null;
 
-	/** Total line amount without tax in a local currency. */
+	/**
+	 * Total line amount without tax in a local currency.
+	 * @var numeric-string
+	 */
 	#[Map('LineExtensionAmount')]
 	private string $lineExtensionAmount;
 
@@ -84,7 +87,10 @@ class InvoiceLine implements Arrayable
 	#[Map('LineExtensionAmountTaxInclusiveCurr')]
 	private ?string $lineExtensionAmountTaxInclusiveCurr = null;
 
-	/** Total line amount including tax in a local currency. */
+	/**
+	 * Total line amount including tax in a local currency.
+	 * @var numeric-string
+	 */
 	#[Map('LineExtensionAmountTaxInclusive')]
 	private string $lineExtensionAmountTaxInclusive;
 
@@ -120,15 +126,20 @@ class InvoiceLine implements Arrayable
 	#[Map('Item')]
 	private ?Item $item = null;
 
+	/**
+	 * @param numeric-string|BcMath\Number $lineExtensionAmount
+	 * @param numeric-string|BcMath\Number $lineExtensionAmountTaxInclusive
+	 */
 	public function __construct(
-		string $id,
-		string|BcMath\Number $lineExtensionAmount,
-		string|BcMath\Number $lineExtensionAmountTaxInclusive,
-		string|BcMath\Number $lineExtensionTaxAmount,
-		string|BcMath\Number $unitPrice,
-		string|BcMath\Number $unitPriceTaxInclusive,
+		string                $id,
+		string|BcMath\Number  $lineExtensionAmount,
+		string|BcMath\Number  $lineExtensionAmountTaxInclusive,
+		string|BcMath\Number  $lineExtensionTaxAmount,
+		string|BcMath\Number  $unitPrice,
+		string|BcMath\Number  $unitPriceTaxInclusive,
 		ClassifiedTaxCategory $classifiedTaxCategory,
-	) {
+	)
+	{
 		$this->setId($id);
 		$this->setLineExtensionAmount($lineExtensionAmount);
 		$this->setLineExtensionAmountTaxInclusive($lineExtensionAmountTaxInclusive);
@@ -246,13 +257,19 @@ class InvoiceLine implements Arrayable
 		return $this;
 	}
 
-	/** @deprecated Method accessors are deprecated, use {@see $lineExtensionAmount} property instead. */
+	/**
+	 * @deprecated Method accessors are deprecated, use {@see $lineExtensionAmount} property instead.
+	 * @return numeric-string
+	 */
 	public function getLineExtensionAmount(): string
 	{
 		return $this->lineExtensionAmount;
 	}
 
-	/** @deprecated Method accessors are deprecated, use {@see $lineExtensionAmount} property instead. */
+	/**
+	 * @deprecated Method accessors are deprecated, use {@see $lineExtensionAmount} property instead.
+	 * @param numeric-string|BcMath\Number $lineExtensionAmount
+	 */
 	public function setLineExtensionAmount(string|BcMath\Number $lineExtensionAmount): self
 	{
 		Deprecations::number($lineExtensionAmount);
@@ -298,13 +315,19 @@ class InvoiceLine implements Arrayable
 		return $this;
 	}
 
-	/** @deprecated Method accessors are deprecated, use {@see $lineExtensionAmountTaxInclusive} property instead. */
+	/**
+	 * @deprecated Method accessors are deprecated, use {@see $lineExtensionAmountTaxInclusive} property instead.
+	 * @return numeric-string
+	 */
 	public function getLineExtensionAmountTaxInclusive(): string
 	{
 		return $this->lineExtensionAmountTaxInclusive;
 	}
 
-	/** @deprecated Method accessors are deprecated, use {@see $lineExtensionAmountTaxInclusive} property instead. */
+	/**
+	 * @deprecated Method accessors are deprecated, use {@see $lineExtensionAmountTaxInclusive} property instead.
+	 * @param numeric-string|BcMath\Number $lineExtensionAmountTaxInclusive
+	 */
 	public function setLineExtensionAmountTaxInclusive(string|BcMath\Number $lineExtensionAmountTaxInclusive): self
 	{
 		Deprecations::number($lineExtensionAmountTaxInclusive);
