Index: src/Schema/Invoice.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/Schema/Invoice.php b/src/Schema/Invoice.php
--- a/src/Schema/Invoice.php	(revision bf8a69c8a828880c891be678a5cd59bdac47596f)
+++ b/src/Schema/Invoice.php	(date 1768532035438)
@@ -37,8 +37,8 @@
  * @property string                                  $refCurrRate
  * @property Invoice\AccountingSupplierParty         $accountingSupplierParty
  * @property Invoice\SellerSupplierParty|null        $sellerSupplierParty
- * @property Invoice\AccountingCustomerParty|null    $accountingCustomerParty
  * @property Invoice\AnonymousCustomerParty|null     $anonymousCustomerParty
+ * @property Invoice\AccountingCustomerParty|null    $accountingCustomerParty
  * @property Invoice\BuyerCustomerParty|null         $buyerCustomerParty
  * @property Invoice\OrderReferences|null            $orderReferences
  * @property Invoice\DeliveryNoteReferences|null     $deliveryNoteReferences
@@ -168,14 +168,14 @@
 	#[Map('SellerSupplierParty')]
 	private ?Invoice\SellerSupplierParty $sellerSupplierParty = null;

+	/** Anonymous receiver of simplified tax document. */
+	#[Map('AnonymousCustomerParty')]
+	private ?Invoice\AnonymousCustomerParty $anonymousCustomerParty = null;
+
 	/** Customer, accounting entity in Commercial Register. */
 	#[Map('AccountingCustomerParty')]
 	private ?Invoice\AccountingCustomerParty $accountingCustomerParty = null;

-	/** Anonymous receiver of simplified tax document. */
-	#[Map('AnonymousCustomerParty')]
-	private ?Invoice\AnonymousCustomerParty $anonymousCustomerParty = null;
-
 	/** Purchaser, invoicing address. */
 	#[Map('BuyerCustomerParty')]
 	private ?Invoice\BuyerCustomerParty $buyerCustomerParty = null;
@@ -553,17 +553,6 @@
 		$this->sellerSupplierParty = $sellerSupplierParty;
 		return $this;
 	}
-
-	public function getAccountingCustomerParty(): ?Invoice\AccountingCustomerParty
-	{
-		return $this->accountingCustomerParty;
-	}
-
-	public function setAccountingCustomerParty(?Invoice\AccountingCustomerParty $accountingCustomerParty): self
-	{
-		$this->accountingCustomerParty = $accountingCustomerParty;
-		return $this;
-	}

 	public function getAnonymousCustomerParty(): ?Invoice\AnonymousCustomerParty
 	{
@@ -575,6 +564,17 @@
 		$this->anonymousCustomerParty = $anonymousCustomerParty;
 		return $this;
 	}
+
+	public function getAccountingCustomerParty(): ?Invoice\AccountingCustomerParty
+	{
+		return $this->accountingCustomerParty;
+	}
+
+	public function setAccountingCustomerParty(?Invoice\AccountingCustomerParty $accountingCustomerParty): self
+	{
+		$this->accountingCustomerParty = $accountingCustomerParty;
+		return $this;
+	}

 	public function getBuyerCustomerParty(): ?Invoice\BuyerCustomerParty
 	{
