diff --git a/src/Schema/Invoice.php b/src/Schema/Invoice.php
index 9c7e1a0..88907fd 100644
--- a/src/Schema/Invoice.php
+++ b/src/Schema/Invoice.php
@@ -37,8 +37,8 @@ use Nette\SmartObject;
  * @property string|BcMath\Number                    $refCurrRate
  * @property Invoice\AccountingSupplierParty         $accountingSupplierParty
  * @property Invoice\SellerSupplierParty|null        $sellerSupplierParty
- * @property Invoice\AccountingCustomerParty|null    $accountingCustomerParty
  * @property Invoice\AnonymousCustomerParty|null     $anonymousCustomerParty
+ * @property Invoice\AccountingCustomerParty|null    $accountingCustomerParty
  * @property Invoice\BuyerCustomerParty|null         $buyerCustomerParty
  * @property Invoice\OrderReferences|null            $orderReferences
  * @property Invoice\DeliveryNoteReferences|null     $deliveryNoteReferences
@@ -168,14 +168,14 @@ class Invoice implements Arrayable
 	#[Map('SellerSupplierParty')]
 	private ?Invoice\SellerSupplierParty $sellerSupplierParty = null;
 
-	/** Customer, accounting entity in Commercial Register. */
-	#[Map('AccountingCustomerParty')]
-	private ?Invoice\AccountingCustomerParty $accountingCustomerParty = null;
-
 	/** Anonymous receiver of simplified tax document. */
 	#[Map('AnonymousCustomerParty')]
 	private ?Invoice\AnonymousCustomerParty $anonymousCustomerParty = null;
 
+	/** Customer, accounting entity in Commercial Register. */
+	#[Map('AccountingCustomerParty')]
+	private ?Invoice\AccountingCustomerParty $accountingCustomerParty = null;
+
 	/** Purchaser, invoicing address. */
 	#[Map('BuyerCustomerParty')]
 	private ?Invoice\BuyerCustomerParty $buyerCustomerParty = null;
@@ -604,29 +604,29 @@ class Invoice implements Arrayable
 		return $this;
 	}
 
-	/** @deprecated Method accessors are deprecated, use {@see $accountingCustomerParty} property instead. */
-	public function getAccountingCustomerParty(): ?Invoice\AccountingCustomerParty
+	/** @deprecated Method accessors are deprecated, use {@see $anonymousCustomerParty} property instead. */
+	public function getAnonymousCustomerParty(): ?Invoice\AnonymousCustomerParty
 	{
-		return $this->accountingCustomerParty;
+		return $this->anonymousCustomerParty;
 	}
 
-	/** @deprecated Method accessors are deprecated, use {@see $accountingCustomerParty} property instead. */
-	public function setAccountingCustomerParty(?Invoice\AccountingCustomerParty $accountingCustomerParty): self
+	/** @deprecated Method accessors are deprecated, use {@see $anonymousCustomerParty} property instead. */
+	public function setAnonymousCustomerParty(?Invoice\AnonymousCustomerParty $anonymousCustomerParty): self
 	{
-		$this->accountingCustomerParty = $accountingCustomerParty;
+		$this->anonymousCustomerParty = $anonymousCustomerParty;
 		return $this;
 	}
 
-	/** @deprecated Method accessors are deprecated, use {@see $anonymousCustomerParty} property instead. */
-	public function getAnonymousCustomerParty(): ?Invoice\AnonymousCustomerParty
+	/** @deprecated Method accessors are deprecated, use {@see $accountingCustomerParty} property instead. */
+	public function getAccountingCustomerParty(): ?Invoice\AccountingCustomerParty
 	{
-		return $this->anonymousCustomerParty;
+		return $this->accountingCustomerParty;
 	}
 
-	/** @deprecated Method accessors are deprecated, use {@see $anonymousCustomerParty} property instead. */
-	public function setAnonymousCustomerParty(?Invoice\AnonymousCustomerParty $anonymousCustomerParty): self
+	/** @deprecated Method accessors are deprecated, use {@see $accountingCustomerParty} property instead. */
+	public function setAccountingCustomerParty(?Invoice\AccountingCustomerParty $accountingCustomerParty): self
 	{
-		$this->anonymousCustomerParty = $anonymousCustomerParty;
+		$this->accountingCustomerParty = $accountingCustomerParty;
 		return $this;
 	}
 
